fixtures/accelerated-peft-autogptq-sample-configuration.yaml

```diff
@@ -0,0 +1,24 @@
+# FMS Acceleration Plugin Configuration. 
+#
+# Each stanza incorporates various configurations for 
+# different fine-tuning / training tasks.
+plugins:
+  # PEFT-related acceleration
+  peft:
+
+    # quantization-releated acceleration
+    # e.g., kernels for quantized base weights
+    quantization:
+
+      # AutoGPTQ quantized base weights.
+      auto_gptq:
+
+        # Kernel to be used for GPTQ linear laeyer
+        # NOTE: Not all kernels are suitable for PEFT training; need to use 
+        # kernels that support autograd forward / backward. The best 
+        # recommendation at the moment is "triton_v2".
+        kernel: triton_v2
+
+        # If true, then will already expect quantized checkpoint 
+        # passed into TrainingArguments.model_name_or_path
+        from_quantized: true
```

Comment:
will from_quantized: false, even work ? it is not supported currently right? so wondering if we should remove this as option in YAML and harcode code True, otherwise throw error?

Reply:
yes, this is a flag supported in the AutoGPTQ package, such that you can take a non-quantized model, and quantize it, then start training. 

But we disable this and we actually do have a [check in our plugin](https://github.com/foundation-model-stack/fms-acceleration/blob/79bc89b3ce15d804e826d7c98e11bde0182fa7a4/plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py#L47), where this will raise an [AccelerationPluginConfigError](https://github.com/foundation-model-stack/fms-acceleration/blob/79bc89b3ce15d804e826d7c98e11bde0182fa7a4/plugins/framework/src/fms_acceleration/framework_plugin.py#L183)

But yes im tihking as we implement the User facing dataclass, I will hide this argument, and only have it in the `AccelerationFrameworkConfig` dataclass (but also hardcode True)

When we have the dataclass I will remove all the YAMLs.

Reply:
as shown in https://github.com/foundation-model-stack/fms-hf-tuning/pull/157#:~:text=%E2%80%A6-,9580b99,-fmt%20%2B%20lint for all non supported cases we will have multi-levels of checking
- checking in the dataclass
- checking in the framework itself

I will update the tests to clarify these checks.